frontend/src/components/uploadedFileDisplay.module.css

```diff
@@ -0,0 +1,29 @@
+.uploadedFileContainer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 36px;
+  margin-bottom: 4px;
+  box-sizing: border-box;
+}
+
```

Comment:
I noticed some extra spacing between the inputContainer and the uploadedFileContainer which I think can be easily fixed by removing the margin from the inputContainer

Reply:
Done :)